src/sentry/conf/server.py

```diff
@@ -2421,6 +2421,24 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
             "platform": "linux/amd64",
         }
     ),
+    "taskbroker": lambda settings, options: (
+        {
+            "image": "ghcr.io/getsentry/taskbroker:latest",
+            "ports": {"50051/tcp": 50051},
+            "environment": {
+                "TASKBROKER_KAFKA_CLUSTER": (
+                    "kafka-kafka-1"
+                    if os.environ.get("USE_NEW_DEVSERVICES") == "1"
+                    else "sentry_kafka"
+                ),
+            },
+            "only_if": options.get("taskbroker.enabled"),
+            # we don't build linux/arm64 snuba images anymore
+            # apple silicon users should have working emulation under colima 0.6.2
+            # or docker desktop
+            "platform": "linux/amd64",
+        }
+    ),
```

Comment:
can you add this to new devservices as well? cc @hubertdeng123 @IanWoodard 

this is the old one which is going away soon!

Reply:
https://github.com/getsentry/sentry/pull/82244